src/com/gfredericks/test/chuck/properties.cljc

```diff
@@ -1,9 +1,10 @@
 (ns com.gfredericks.test.chuck.properties
   "Alternative to clojure.test.check.properties."
   (:require [clojure.set :as sets]
-            [clojure.test.check.generators :as gen]
-            [clojure.test.check.properties :as prop]
-            [com.gfredericks.test.chuck.generators :as gen']))
+            [clojure.test.check.properties :as prop
+             #?@(:cljs [:include-macros true])]
+            [com.gfredericks.test.chuck.generators :as gen
```

Comment:
I'd like to keep the alias here as `gen'`; I use `gen` for `clojure.test.check.generators`.